src/Patronage.Api/MediatR/BoardStatus/Commands/CreateBoardStatusCommand.cs

```diff
@@ -1,7 +1,15 @@
 ﻿using MediatR;
-using Patronage.Contracts.ModelDtos;
+using Patronage.Contracts.ModelDtos.BoardsStatus;
 
 namespace Patronage.Api.MediatR.BoardStatus.Commands
 {
-    public record CreateBoardStatusCommand(BoardStatusDto Dto) : IRequest<bool>;
+    public class CreateBoardStatusCommand : IRequest<bool>
+    {
+        public BoardStatusDto Dto { get; set; } = null!;
```

Comment:
```suggestion
        public BoardStatusDto Dto { get; set; }
```
null isn't needed if we always set the Dto in constructor

Reply:
Okey, i will delete this.